requirements.txt

```diff
@@ -163,13 +166,14 @@ soupsieve==1.9.3
 SQLAlchemy==1.3.8
 sshpubkeys==3.1.0
 stevedore==1.31.0
-stringscore==0.1.0
+#stringscore==0.1.0
 stripe==2.36.2
 supervisor-logging==0.0.9
 supervisor-stdout==0.1.1
 supervisor==4.0.4
 tldextract==2.2.1
 toposort==1.5
+trollius #==2.2.post1
```

Comment:
I would make a comment that this is a backport of asyncio (a native feature in Python3) and that it can probably be removed. In my previous PR, I changed all references to trollius to asyncio in the Python code.